submissions/serializers.py

```diff
@@ -57,6 +57,29 @@ class TeamSubmissionSerializer(serializers.ModelSerializer):
     submitted_at = DateTimeField(format=None, required=False)
     created_at = DateTimeField(source='created', format=None, required=False)
     attempt_number = IntegerField(min_value=0)
+    # Prevent Django Rest Framework from converting the answer (dict or str)
+    # to a string.
+    answer = serializers.SerializerMethodField()
+
+    def get_answer(self, obj):  # pylint: disable=unused-argument
```

Comment:
I don't think there's anything wrong with passing in the context dict, but if we want to avoid that I think we could also do 

`return obj.submissions.first().answer`

Reply:
Cool. Didn't realize that was an option either. I will use context if no major objections.